src/components/Draft.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState, useContext } from "react";
+import { TextField, Box, Stack, Button } from "@mui/material";
+import { Sign } from '../util'
+import { ApplicationContext } from '../App';
+import SendIcon from '@mui/icons-material/Send';
+
+export function Draft(props: any) {
```

Comment:
could you define Draft Interface Type like other components please

Reply:
I've updated the code according to the requested change.